admin/controllers/school.js

```diff
@@ -368,18 +372,83 @@ const getSelectPupilNotTakingCheck = async (req, res, next) => {
 
   let attendanceCodes
   let formData
+  let pupilsList
+  let htmlSortDirection = []
+  let arrowSortDirection = []
 
+  // Sorting
+  const sortField = req.params.sortField === undefined ? 'name' : req.params.sortField
+  const sortDirection = req.params.sortDirection === undefined ? 'asc' : req.params.sortDirection
+
+  let sortingDirection = [
+    {
+      'key': 'name',
+      'value': 'asc'
+    },
+    {
+      'key': 'reason',
+      'value': 'asc'
+    }
+  ]
+
+  // Markup links and arrows
+  sortingDirection.map((sd, index) => {
+    if (sd.key === sortField) {
+      htmlSortDirection[sd.key] = (sortDirection === 'asc' ? 'desc' : 'asc')
```

Comment:
no need the brackets around the turnary expression

Reply:
I know, I like them, they make the code more readable.

Reply:
Parenthesis removed everywhere.